dev-docs/conventions.md

```diff
@@ -1,39 +1,11 @@
-# Process conventions
+# Writing to customers: style policy
 
-## Pull request process
+Whenever we write text facing the customer (e.g docs, warnings, errors), we follow the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/welcome/).
```

Comment:
```suggestion
Whenever you write text facing the customer (e.g, docs, warnings, errors), follow the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/welcome/).
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we